### PR TITLE
Add Cassandra scaler

### PR DESCRIPTION
**Pull request by nilayasiktoprak**
Signed-off-by: nilayasiktoprak <nilayasiktoprak@gmail.com>

Add a new Cassandra scaler.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [ ] Tests have been added
- [ ] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [x] Changelog has been updated

Relates to #1738

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```
Comment (zroubalik) [inline config/manager/kustomization.yaml:5]: Please exclude this file from your PR.

File: config/metrics-server/kustomization.yaml [hunk 2]
```diff
@@ -10,5 +10,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```
Comment (zroubalik) [inline config/metrics-server/kustomization.yaml:10]: Please exclude this file from your PR.

**Review by zroubalik — COMMENTED** [review]
Thank you for you contribution! Is this PR ready for a review? I haven't yet check the code properly, have some input though:
- could you please add unit test as it is done for the rest of the scalers (`cassandar_scaler_test.go)?
- could you please add e2e test for this scaler? (in this case probably an ephemeral deployment of Cassandra on k8s and checking that applied ScaledObject scales some target deployment (similar as it is implemented in e2e tests for other scalers)  https://github.com/kedacore/keda/tree/main/tests

**Comment by nilayasiktoprak** [comment]
I will add them as soon as possible :) And will delete those 2 files, too. Thank you for the review!

**Comment by nilayasiktoprak** [comment]
This PR is moved to #1841

**Pull request closed without merging**